### ISSUE TITLE
Proposed slider design

### ISSUE THREAD

**Issue by tifftea** [carbon-design-system/carbon#5925]
#### I am submitting a...

- [ x] Feature request
- [ ] Design defect
- [ ] Source code defect
- [ ] Demo/documentation defect
- [x ] Other (design proposal)

### Issue description

When it comes to scaling DB2’s compute and storage, we aim to solve three problems:
1) convey threshold
2) provide recommendation
3) ability to scale up and down

The current slider provided by Carbon is too simple and does not provide enough value to our users.

In addition, we created custom flyovers that not only enables value change as the user drags the handles, but the flyovers give users incentives to scale up/down.
The use case of the slider for scaling is to nudge users explore multiple options and the value associated with each pricing point and, hence, aid them in making an informed decision.
The requirement of the slider are:
- display pricing options as the visual feedback as the users hover over the product areas
- display the price continually, and the slider is never blocked by price changes.
- indicate the recommendations that is accessibility-compliant. 

Use cases:
Slider 1: Based on the system performance and resource consumption, the additional core is suggested.
Slider 2: The user increases the storage and decreases the core. The price per month updates accordingly. The updates activates the Save button. 
Slider 3: The user increases both storage and cores for optimal performance.

The CX component on left lacks the capabilities to satisfy the use cases.

Before:
![carbon slider](https://user-images.githubusercontent.com/63752979/80133148-a11b7d80-8551-11ea-81c5-7d112b2c24b0.png)

Proposed design:
<img width="703" alt="scaling sliders" src="https://user-images.githubusercontent.com/63752979/80133186-b2fd2080-8551-11ea-8c5f-1e850566a5ec.png">

**Comment by joshblack** [comment]
Closing as duplicate of: https://github.com/carbon-design-system/carbon/issues/5924
